content/08-cardano-sidechains/03-example-evm-sidechain/03-metamask.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Working with MetaMask
+metaTitle: Working with MetaMask
+---
+
+>*MetaMask compatibility with the sidechain is officially tested with Google Chrome and Brave web browsers.*
+
+## Install MetaMask
+
+Use this link to install MetaMask as a browser extension or mobile app: https://metamask.io/.
+
+![install-meta-mask](https://user-images.githubusercontent.com/10556209/148433190-2aabc719-67fb-40f3-a459-0f5f0d81e169.png)
+
+## Create or import account
+
+Choose the option to create an account.
+
+![create-account](../../images/sc-metammask-create-account.png)  
+Back up your wallet seed phrase and store it in a safe place.
+
+## Add Cardano EVM sidechain network(s) in settings
+
+Settings > Networks > Add Network > Input the following settings and save.
+
+- Network Name: `EVM Sidechain Tesnet`
+- New RPC URL: `<PENDING_URL>`
+- Chain ID: `78`
+- Currency Symbol: `SC_Token`<sup>1</sup>
+- Block Explorer URL: `<PENDING_URL>`
+
+Now you can select the Cardano EVM sidechain network from MetaMask:
+
+![select-network](../../images/sc-metamask-select-testnet.png)
+
+## Known issues
+
+- If the sidechain network was added correctly, but MetaMask does not send transactions, update balance, or simply not interact with the chain, then try to [Reset your MetaMask account](https://metamask.zendesk.com/hc/en-us/articles/360015488891-How-to-reset-an-account).
+- MetaMask compatibility with the sidechain is officially tested with Google Chrome and Brave web browsers. To avoid browser compatbility issues, use Google Chrome or Brave.
```

Comment:
The team has tested MetaMask compatibility with the sidechain using Google Chrome and Brave web browsers. To avoid browser compatibility issues, use the same browsers.